prowler/providers/aws/services/elasticache/elasticache_redis_cluster_auth_enabled/elasticache_redis_cluster_auth_enabled.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticache_redis_cluster_auth_enabled",
```

Comment:
I don't understand why names `redis_cluster` if you are giving findings about replication groups

Reply:
I handled it this way because all replication group checks were done that way. Making these corrections would imply renaming the checkID and outputs of all other replication group checks as well.